lib/load_brocfile.js

```diff
@@ -4,23 +4,71 @@ const path = require('path');
 const findup = require('findup-sync');
 const esmRequire = require('esm')(module);
 
+/**
+ * Require a brocfile via either ESM or TypeScript
+ *
+ * @param {String} brocfilePath The path to the brocfile
+ * @returns {*}
+ */
+function requireBrocfile(brocfilePath) {
+  let brocfile;
+
+  if (brocfilePath.match(/\.ts$/)) {
+    const configPath = findup('tsconfig.json', {
+      nocase: true,
+    });
+
+    let options;
+
+    if (configPath) {
+      options = { project: configPath };
+    } else {
+      options = {
+        // Define default compilerOptions
+        compilerOptions: {
```

Comment:
Looking for a steer on these defaults

Reply:
These seem reasonable; the only one I'd consider changing is `target` – I'd set it to whatever the current Node LTS supports (which IIRC is ES2015? Might already be 2016).

Reply:
I think it should be node 6, which is what's in the package.json as the min node version. Agree?

Reply:
Seems 👍 to me!